beamer/tests/agent/test_cli.py

```diff
@@ -38,6 +40,23 @@ def _generate_deployment_dir(output_dir, root, contracts):
     shutil.copy(src / "ResolutionRegistry.json", output_dir)
 
 
+@pytest.fixture
+def setup_relayer_executable():
+    target = pathlib.Path(__file__).parent.parent.parent.joinpath("data/relayers/")
+    target.mkdir(parents=True, exist_ok=True)
+
+    executable = target / _RELAYER_NAMES[sys.platform]
+    if executable.exists():
+        return
+
+    executable.write_text("text")
+
+    yield
+
+    executable.unlink()
+
```

Comment:
Nitpick: I was thinking more along the lines of

```python
 def setup_relayer_executable():
    path = beamer.l1_resolution.get_relayer_path()
    if path.exists():
       return

    path.parent.mkdir(parents=True, exist_ok=True)
    path.write_text("")
    yield
    path.unlink()
```
where we don't need to know the explicit location of the relayer nor the platform stuff. This would also
avoid importing private items.